py/server/deephaven/table_listener.py

```diff
@@ -237,25 +238,29 @@ def modified_columns(self) -> List[str]:
         return list(cols) if cols else []
 
 
-def _do_locked(f: Callable, lock_type="shared") -> None:
-    """Executes a function while holding the UpdateGraphProcessor (UGP) lock.  Holding the UGP lock
+def _do_locked(ug: Union[_JUpdateGraph, Table], f: Callable, lock_type="shared") -> None:
```

Comment:
Two problems:
1) The API is exposing Java objects to the user.
2) The type hint is referring to a private variable.